recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "5.8" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```

Comment:
Normally we do `{{ name }}-{{ version }}.tar.gz`. This is a bit nicer for people building the recipes locally as it allows `conda-build`'s source cache to keep track of `rocksdb` tarballs independently from those of other tarballs that may share the same version number.